frontend/js/views/main.js

```diff
@@ -805,6 +807,8 @@ define(["jquery",
              * @alias module:views-main.MainView#onWindowResize
              */
             onWindowResize: function () {
```

Comment:
It would be nice to debounce "onWindowResize" using underscore's [debounce](https://github.com/jashkenas/underscore/blob/1.8.3/underscore.js#L813) function to prevent sluggish behaviour.